plugin-management-cli/src/test/java/io/jenkins/tools/pluginmanager/cli/CliOptionsTest.java

```diff
@@ -0,0 +1,161 @@
+package io.jenkins.tools.pluginmanager.cli;
+
+import io.jenkins.tools.pluginmanager.config.Config;
+import io.jenkins.tools.pluginmanager.config.Settings;
+import java.io.File;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({CliOptions.class, System.class})
+public class CliOptionsTest {
+    private CliOptions options;
+    private CmdLineParser parser;
+
+    @Before
+    public void createParser() throws CmdLineException {
+        options = new CliOptions();
+        parser = new CmdLineParser(options);
+    }
+
+
+    @Test
+    public void setupDefaultsTest() throws CmdLineException {
+        parser.parseArgument();
+
+        PowerMockito.mockStatic(System.class);
+
+        Mockito.when(System.getenv("JENKINS_UC")).thenReturn("");
```

Comment:
static import mockito for fluent testing:
```suggestion
       when(System.getenv("JENKINS_UC")).thenReturn("");
```